src/ansys/grantami/bomanalytics/_query_results.py

```diff
@@ -891,16 +925,82 @@ def process(self) -> SustainabilityPhaseSummaryResult:
         """
         return self._process_summary
 
+    @property
+    def distribution_transport_summary(self) -> TransportSummaryByCategoryResult | None:
```

Comment:
Since there are only ever two categories in a result, it makes sense to define two properties which each return either a result or None if using an old version of the API. This is different to the summary by job, which has a variable number of results, and so returns a list. In that case it always returns an empty list, but the list may be empty.